__tests__/app.test.js

```diff
@@ -44,4 +43,44 @@ describe("GET /api/topics", () => {
         })
       })
   })
+})
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an article object each with the correct properties", () => {
+    return request(app)
+      .get('/api/articles/1')  
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject
+        ({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
+          })
+       
+      })
+  })
+
+  test("404: if articles doesnt exist, respond with error", () => {
```

Comment:
I'd be even more specific with your test description to say it's a valid but non-existent article id :)